csunplugged/topics/content/en/image-representation/unit-plan/unit-plan.md

```diff
@@ -0,0 +1,94 @@
+
+{image file-path="img/topics/binary-picture-showing-bits.png" alt="If we look closely at an image on a computer screen we can see it is made up of a grid of tiny dots called pixels. The colour of each of these is stored inside a computer using binary."}
+
+The type of information a set of digits **represents** changes the way we interpret it.
+For example, if a set of digits represents an image then we interpret those digits as the colours of different pixels.
+If it is meant to be text on the other hand, we interpret it as letters.
+The ability for computers to represent multiple types of information using just two states (represented with binary digits) is one of the things that makes them so powerful.
+
+Just like learning about the binary number system, exploring image representation exposes students to the Computational Thinking concepts of Abstraction and Decomposition.
+Students learn to break down images into pixels and then to digits, and how to move back up from digits, to pixels, to images.
+
+## Digital Technologies | Algorithms
+
+When pixels are used to represent images, the process we go through to convert digits into these pixels is a type of Algorithm.
+There are many different algorithms computers use to do this, and the one they use depends on things like the file type of the images being displayed, for example if it is a picture like a .png or .jpeg, or a video file like an .mp4 or .avi; or what resolution the image is.
+We also use Algorithms to compress images to smaller file sizes, which means they use less memory, can be processed more quickly, and are faster to download.
+Through lerning about image representation students practice algorithmic thinking, pattern matching, and abstract thinking.
+
+## Vocabulary Explained
+
+### Pixel
+
+Computer screens are divided into a grid of tiny coloured squares, which are called pixels.
+The colour of each pixel can be set by a computer and they are used to display images on a computer screen (and printed images).
+The word pixel is an abbreviation of *picture element*.
+
+### Display Resolution
+
+The resolution of a screen refers to the number, or density, of pixels on the screen.
+It is usually defined as either the number of pixels per inch on the screen, or the width and height of the screen measured in pixels (e.g. 1920 by 1080 pixels).
+
+## Real world implications
+
+The way images are digitally repersented affects how they are displayed, created, stored, and manipulated.
+The more bits we use to store the colour of a pixel, the more different colours we can make.
+If we use one bit to store the colour of a pixel then we only have two options for what that colour can be (remember, with one bit we can represent two different values only, because we can only use 0 or 1).
+If we use 8 bits (1 byte) then we can display 256 different colours instead, and if we use 24 bits (3 bytes) then we can display over 16 million different colours, which is more than the human eye can see!
```

Comment:
You've avoided using 'bit' the entire document and now it's used 5 times in a paragraph, what has changed?

Reply:
I used 'bit' in this part because it relates to the activities/lessons and how we use a limited number of bits in each activity to represent colours, e.g. when they are colouring the black and white image there are only two different values that can appear in a square - 0 or 1. For the grey scale images there are two bits, so they can do 4 colours (black, grey, light grey, white), and the coloured images have 5 bits (they just get a limited number of the colours though).
Does that make sense? The idea was to reinforce that how we represent the information changes what we can do with it.

I'll go back and review the rest of the unit plan and see if I should be referring to bits in other places as well.

Reply:
Okay that makes sense, perhaps just reinforce the link between bit and binary digit (e.g. add `(bit)` the first time 'binary digit' is mentioned). Reading through it I had the impression 'bit' was being avoided - perhaps because the term hadn't been taught yet - and when it was suddenly used a lot I found that quite jarring

Reply:
Good point, I've added in (bit) and (binary digit) in a couple of appropriate places to help with that reinforcement.
Before starting this topic they are meant to do the first binary numbers lesson so they should be at least a little familiar with the terms, but having the reinforcement in this is probably still necessary.